test/unit/math/mix/mat/prob/std_binormal_lcdf_test.cpp

```diff
@@ -0,0 +1,1245 @@
+#include <stan/math/mix/mat.hpp>
+#include <gtest/gtest.h>
+#include <test/unit/math/rev/mat/fun/util.hpp>
+#include <test/unit/math/rev/mat/util.hpp>
+#include <limits>
+#include <vector>
+
+using Eigen::Dynamic;
+using Eigen::Matrix;
+using stan::math::fvar;
+using stan::math::var;
+using std::vector;
+
+// Vector, real
+template <typename T>
+void to_function_input(int N_y,
+                       const Eigen::Matrix<T, Eigen::Dynamic, 1>& inp_vec,
+                       Eigen::Matrix<T, Eigen::Dynamic, 1>& y, T& rho) {
+  y(0) = inp_vec(0);
+  y(1) = inp_vec(1);
+  rho = T(inp_vec(2));
```

Comment:
```
rho = inp_vec(2);
```
The promotion will happen automatically unless `T` has a contsructor inconsistent with assignment.